src/main/resources/sscs-non-working-days-england-and-wales.json

```diff
@@ -0,0 +1,45 @@
+{
+  "events": [
+    {
+      "title": "New Year’s Day",
+      "date": "2023-01-02",
```

Comment:
Is there a dependency on updating these values or is it more a default file for dmn tables to pull in?

Reply:
These files will need to be updated annually, I have brought this up with the BA. If the dates are always going to be standard bank holidays plus privilege days we could use the standard calendar for bank holidays with a custom calendar for privilege days.